src/wp-includes/default-filters.php

```diff
@@ -195,16 +195,17 @@
 add_filter( 'the_content', 'wpautop' );
 add_filter( 'the_content', 'shortcode_unautop' );
 add_filter( 'the_content', 'prepend_attachment' );
-add_filter( 'the_content', 'wp_filter_content_tags' );
 add_filter( 'the_content', 'wp_replace_insecure_home_url' );
+add_filter( 'the_content', 'do_shortcode', 11 ); // AFTER wpautop().
+add_filter( 'the_content', 'wp_filter_content_tags', 12 ); // Runs after do_shortcode().
```

Comment:
If a developer have unhooked on priority 10 if they don't want image features. Looking online, I can't see anyone doing this, but it worth noting.

Reply:
Thanks @spacedmonkey, see https://github.com/WordPress/wordpress-develop/pull/4973#discussion_r1320204338. I did a bunch of research already, and this change should be safe to make. Needless to say there will be a dev note, but it doesn't look like this is going to be a big problem since there is zero usage in the plugin repository.